app/uk/gov/hmrc/soletraderidentificationfrontend/models/JourneyConfig.scala

```diff
@@ -25,20 +25,23 @@ object JourneyConfig {
   private val optServiceNameKey = "optServiceName"
   private val deskProServiceIdKey = "deskProServiceId"
   private val signOutUrlKey = "signOutUrl"
+  private val enableSautrCheckKey = "false"
 
   implicit val format: OFormat[JourneyConfig] = new OFormat[JourneyConfig] {
     override def reads(json: JsValue): JsResult[JourneyConfig] = for {
       continueUrl <- (json \ continueUrlKey).validate[String]
       optServiceName <- (json \ optServiceNameKey).validateOpt[String]
       deskProServiceId <- (json \ deskProServiceIdKey).validate[String]
       signOutUrl <- (json \ signOutUrlKey).validate[String]
-    } yield JourneyConfig(continueUrl, PageConfig(optServiceName, deskProServiceId, signOutUrl))
+      enableSautrCheck <- (json \ enableSautrCheckKey).validateOpt[Boolean]
```

Comment:
There may be issues with the reads using a default parameter. There's potentially a couple fixes